.gitignore

```diff
@@ -1 +1,4 @@
 node_modules/
+.parcel-cache/
+dist/
+package-lock.json
```

Comment:
Good to put these Parcel-specific patterns inside the sandboxes, where theu are needed. That way, they are included in forked sandboxes.

Reply:
I don't think the sandboxes are themselves git repos when used 🤔 
but I can add a gitignore to each anyway 🤷‍♂️